etc/openwrt/openthread-br/README.md

```diff
@@ -0,0 +1,44 @@
+OpenThread Border Router on OpenWRT
+===================================
+
+## Get Started
+
+This is for local development.
+
+1. Add OpenThread feed.
+
+Suppose `OPENWRT_TOP_SRCDIR` is root of openwrt sources.
+
+```bash
+echo src-link openthread "$(pwd)/etc/openwrt" >> ${OPENWRT_TOP_SRCDIR}/feeds.conf
+cd "${OPENWRT_TOP_SRCDIR}"
+./scripts/feeds update openthread
+./scripts/feeds install openthread-br
+```
+2. Enable OpenThread.
+
+```bash
+make menuconfig
+```
+
+Check *Network* > *OpenThread Border Router*.
+
+3. Build OpenThread Border Router
+
+```bash
+make package/openthread/compile
+```
+
+or verbose make,
```

Comment:
Why would the user want to do a verbose make? Can we explain?  Otherwise we could just pick one.

Reply:
I wrote this mainly for developers. After we publish the package to OpenWRT's  official feeds. Normal users just need to install by:

```bash
opkg install openthread-br
```